src/controllers/dynakube/activegate/internal/statefulset/statefulset.go

```diff
@@ -150,7 +178,7 @@ func (statefulSetBuilder Builder) buildBaseContainer() []corev1.Container {
 }
 
 func (statefulSetBuilder Builder) buildResources() corev1.ResourceRequirements {
-	if statefulSetBuilder.dynakube.IsSyntheticActiveGateEnabled() {
```

Comment:
else is not needed here, you can directly return without and else statement after the if